src/js/jsx/DocumentHeader.jsx

```diff
@@ -57,6 +59,26 @@ define(function (require, exports, module) {
             };
         },
 
+        /**
+         * Update the sizes of the panels.
+         *
+         * @private
+         */
+        _updatePanelSizes: function () {
+            var node = React.findDOMNode(this.refs.tabContainer),
+                headerHeight;
+
+            if (node) {
+                headerHeight = node.getBoundingClientRect().height;
```

Comment:
Yes, but it's not accurate. It doesn't get the border, which adds non-zero width. This change was intentional. Also, since we're not trying to animate with this in a tight loop, I don't think the speed difference will be measurable.